tutorials/ExploringDeutschJozsaAlgorithm/ReferenceImplementation.qs

```diff
@@ -30,8 +30,8 @@ namespace Quantum.Kata.DeutschJozsaAlgorithm {
     function IsFunctionConstant_Classical_Reference (N : Int, f : (Int -> Int)) : Bool {
         // Get the first value of the function
         let firstValue = f(0);
-        // Try all the following inputs to see if any of the values differ
-        for input in 1 .. 2 <<< (N - 1) {
+        // Try all the following inputs to see if any of the values differ; should be run 2ᴺ⁻¹ + 1 times
```

Comment:
A small fix in the comment: the loop actually runs 2ᴺ⁻¹ times - we sample 2ᴺ⁻¹+1 values but we got f(0) outside of the loop, so the loop goes through one less iteration than the total number of values we need
```suggestion
        // Try all the following inputs to see if any of the values differ; should be run 2ᴺ⁻¹ times
```